camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java

```diff
@@ -62,7 +63,12 @@ public void init(Properties props, String topicName) {
     String payloadString;
     JsonObject jsonObject;
 
-    payloadString = new String(payload);
```

Comment:
Does Kafka always encode bytes in UTF-8?  Do they configure the encoding in one of the kafka config parameter?